knative-operator/deploy/resources/knativekafka/broker/eventing-kafka-broker-extra.yaml

```diff
@@ -0,0 +1,55 @@
+# ClusterRole to be able to create resources for namespaced broker
+# Kafka controller creates some additional resources, such as ServiceMonitors, in the broker namespace
+# when a namespaced broker is created.
+# The controller will need the permissions listed in this role.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: eventing-kafka-namespaced-broker-additional-resources
```

Comment:
aren't these two files the same?
should we add them on midstream, and continue to just populate this folder with manifests from midstream?

Reply:
I thought about that, but that's will be patch then. ...which is not ideal :(